File: backend/src/main/java/com/api/prontuario/entites/Pessoa.java
package com.api.prontuario.entites;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.Generated;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.GenericGenerator;

import java.util.Date;

@Entity
@Data
@NoArgsConstructor

public class Pessoa {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String cpf;
    private String nome;
    private Date dataNascimento;
    private String telefone;
}

File: frontend/src/app/shared/model/pessoa.ts
export class Pessoa {
  cpf: string;
  nome: string;
  dataNascimento: Date;
  telefone: string;
  id: number;

  constructor(cpf: string, nome: string, dataNascimento: Date, telefone: string, id: number) {
    this.cpf = cpf;
    this.nome = nome;
    this.dataNascimento = dataNascimento;
    this.telefone = telefone;
    this.id = id;
  }
}


